api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/NewCardService.java

```diff
@@ -33,16 +38,18 @@
      * </p>
      *
      * @param claim the claim to create a new card for.
+     * @param datesOfBirthOfChildren dates of birth of the claimant's children
      */
     @SuppressWarnings("PMD.NullAssignment")
     @Transactional
-    public void createNewCard(Claim claim) {
+    public void createNewCard(Claim claim, List<LocalDate> datesOfBirthOfChildren) {
```

Comment:
It feels like passing the dates of birth through all these message calls is quite messy when we could simply load the dates of birth from the PaymentCycle when the MI reporting message is loaded to be processed. @YetAnotherMatt - this has sparked a bit of debate with Dave and I and we can see pros and cons of both, which way do you prefer?

Reply:
I don't think we can do that for a new card, sadly - the payment cycle may not exist yet. If we start storing the full eligibility and entitlement decision on the claim then we could use that.

Reply:
I might be wrong - not able to spend much time on it right now I'm afraid. Will look again tomorrow.

Reply:
I'm sceptical of getting the information later as it creates an order dependency which might forget if we make changes at a later date.
Here we can be 100% sure that we have the necessary information it's being passed in directly.

Reply:
Once we have the decision from DWP then we can be sure of having the children. Could we add a TODO statement to use that once it's available? (HTBHF-2457)

Reply:
Added Todo statement.